Minio/RequestExtensions.cs

```diff
@@ -145,9 +145,13 @@ await requestMessageBuilder.ResponseWriter(responseResult.ContentStream, cancell
                 throw new BucketNotFoundException();
 
             if (responseResult is not null)
+            {
                 responseResult.Exception = ex;
-            else
-                responseResult = new ResponseResult(request, ex);
+                throw;
+            }
+
+            responseResult = new ResponseResult(request, ex);
```

Comment:
This code will never be executed. Because `responseResult` is set at line 93, the previous `if` statement will always be true and the exception will be rethrown.

Reply:
Done!